api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchQueryBuilder.scala

```diff
@@ -42,8 +42,8 @@ object QueryDefaults {
     // TODO: (jamesgorrie) think of a more sustainable way of doing this
     // maybe having a just a list of terms that we use terms queries to query against,
     // and then have more structured data underlying
-    ("data.subjects.*", Some(8.0)),
-    ("data.genres.label", Some(8.0)),
+    ("subjects.agent.concepts.agent.label", Some(8.0)),
+    ("genres.concepts.agent.label", Some(8.0)),
```

Comment:
Is removing `data` a mistake?

Reply:
Thank you - added a test to make sure we do actually search, and rank highly, these fields.